templates/rapidsai/partials/clone_blazing_notebooks.j2

```diff
@@ -0,0 +1,9 @@
+{# This partial clones the "Welcome to BlazingSQL" repo containing example notebooks. #}
```

Comment:
can this file be renamed to `clone_blazing_notebooks.dockerfile.j2` to match the others?